modules/site-icon/jetpack-site-icon.php

```diff
@@ -364,7 +371,7 @@ public static function crop_page() {
 		$crop_ration = $crop_data['large_image_data'][0] / $crop_data['resized_image_data'][0]; // always bigger then 1
 
 		// lets make sure that the Javascript ia also loaded
```

Comment:
spelling mistake on this line. should be `is` not `ia` 

This function should only be registered on the jetpack upload_blavatar_head which should only be something that you can do if WP is not running WP 4.3. 

I don't think we will have to worry about older Jetpack versions conflicting with the WP 4.3 or higher.